docs/src/thermodynamics.md

```diff
@@ -30,14 +30,37 @@ Above, ``R = ℛ / m`` is the specific gas constant given the
 ``ℛ ≈ 8.31\;\mathrm{J} \, \mathrm{K}^{-1} \, \mathrm{mol}^{-1}`` and molar mass ``m`` of the gas species under consideration.
 
 The [first law of thermodynamics](https://en.wikipedia.org/wiki/First_law_of_thermodynamics),
-aka "conservation of energy", states that infinitesimal changes
-in internal energy ``\mathrm{d} ê`` are related to infinitesimal changes
-in temperature ``\mathrm{d} T`` and pressure ``\mathrm{d} p`` according to
+aka "conservation of energy", states that an infinitesimal change in the
+external energy ``\mathrm{d} Q`` are related to infinitesimal changes
+in temperature ``\mathrm{d} T`` and pressure ``\mathrm{d} p`` according to:[^1]
 
 ```math
-\mathrm{d} ê = cᵖ \mathrm{d} T - \frac{\mathrm{d} p}{\rho}
+\mathrm{d} Q = cᵖ \mathrm{d} T - \frac{\mathrm{d} p}{\rho} ,
```

Comment:
Agree with this, except I am worried about notation bc we use Q for heat flux elsewhere